src/models/vtr/classifier.py

```diff
@@ -74,5 +87,12 @@ def forward(self, input_batch: dict[str, torch.Tensor]) -> torch.Tensor:
 
         encoder_output = encoder_output.mean(dim=1)  # batch_size, emb_size
         encoder_output = self.norm(encoder_output)  # batch_size, emb_size
-        result = self.classifier(encoder_output).squeeze(1)  # batch_size
+        logits = self.classifier(encoder_output).squeeze(1)  # batch_size
+
+        result = {"logits": logits}
+
+        # OCR
+        if self.ocr_flag:
+            result["ocr logits"] = self.ocr(conv)
```

Comment:
It is not good to use spaces in such keys, it may affect in the future somewhere. Just replace them with `_`.